health-sdk/sdk/src/main/java/net/gini/android/health/sdk/GiniHealth.kt

```diff
@@ -145,17 +145,43 @@ class GiniHealth(
      * Gini Pay API and verifying that the extraction's payment state is "Payable".
      *
      * @return `true` if the document is payable and `false` otherwise
-     * @throws Exception if there was an error while retrieving the document or the extractions
```

Comment:
[Request] We need to keep the @throws here and also add it to `checkIfDocumentContainsMultipleDocuments()` because `getExtractionsForDocument()` is private and so clients won't know that `checkIfDocumentIsPayable()` and `checkIfDocumentContainsMultipleDocuments()` can throw exceptions.